src/main/java/me/zeroeightsix/kami/util/VectorUtil.java

```diff
@@ -0,0 +1,75 @@
+package me.zeroeightsix.kami.util;
+
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.util.math.Vec3d;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class VectorUtil {
+
+    public static double getDistance(Vec3d vecA, Vec3d vecB) {
+        return MathHelper.sqrt((vecA.x - vecB.x) * (vecA.x - vecB.x) + (vecA.y - vecB.y) * (vecA.y - vecB.y) + (vecA.z - vecB.z) * (vecA.z - vecB.z));
```

Comment:
Perhaps make use of `Math.sqrt` and `Math.pow`

Reply:
yes